docs/utilities/validation.md

```diff
@@ -31,31 +36,22 @@ We support any JSONSchema draft supported by [fastjsonschema](https://horejsek.g
 
 It will fail fast with `SchemaValidationError` exception if event or response doesn't conform with given JSON Schema.
 
-=== "validator_decorator.py"
+=== "getting_started_validator_decorator_function.py"
 
-    ```python hl_lines="3 5"
-    from aws_lambda_powertools.utilities.validation import validator
+	```python hl_lines="8 27 28 42"
+    --8<-- "examples/validation/src/getting_started_validator_decorator_function.py"
+	```
 
-    import schemas
+=== "getting_started_validator_decorator_schema.py"
 
-    @validator(inbound_schema=schemas.INPUT, outbound_schema=schemas.OUTPUT)
-    def handler(event, context):
-        return event
-    ```
+	```python hl_lines="7 8 10 12 17 19 24 26 28 41 42 44 46 51 53"
```

Comment:
I'm also a little concerned about the number of highlights when I was checking the rendered documentation. Can we switch to an example or two, maybe the most important parts?

Reply:
I'm not sure if it's a problem to highlight all these lines in the schema file. The reason for this is to make sure it's very clear to someone who wants to validate input and output data. The types must match 100%, otherwise it will fail.

For example: I highlighted line 24 because if the user doesn't enter a valid ipv4 address, it fails. Same for line 53, which expects an object.

Might we remove highlighting for examples and required fields (lines 7, 8, 41, 42)?